### ISSUE TITLE
Jelly_Builder::load() returns unloaded model when no parameter is supplied

### ISSUE THREAD

**Issue by aron** [jonathangeiger/kohana-jelly#132]
This was created in [7f6cb1](http://github.com/jonathangeiger/kohana-jelly/commit/7f6cb1978c2b3164e6ea9aa5b18cc8e51c6cc350), see ticket #120.

I'm not 100% sure returning a factory object is the correct behavior if no `$key` is passed in. I've been using this method to load a single row. It was working just fine until this update now I'm returned an unloaded model.

For example:

```
$user = Jelly::select('user')->where('name', '=', 'bill')->load();
```

Running this used to return a loaded user object (if the row existed in the DB). Now I get an empty model. Calling `load()` on a query builder _should_ load the object. The old behaviour seems correct to me.

Cheers,

Aron

**Comment by banks** [comment]
Not sure if this was intended or not. Hopefully it will get sorted soon - we are both pretty busy!

**Comment by aron** [comment]
That's fine, I've just forked the repo and updated it here, aron@3ca2a3cf899d9afe998d286f41c357b599c55102. I sent you guys a pull request should you be interested.

Cheers,

Aron

**Comment by jonathangeiger** [comment]
Closed by 72456d507236ab2527473656369e3a39d7adada9. Thanks aron!
